aiida_registry/cli.py

```diff
@@ -22,8 +22,7 @@ def fetch(package):
 @click.option(
     "--container-image",
     # should use aiidateam/aiida-core-with-services:lastest after the version is released
-    # default="aiidateam/aiida-core-with-services:edge",
-    default="aiidateam/aiida-core:latest",
+    default="aiidateam/aiida-core-with-services:latest",
```

Comment:
```suggestion
    default="ghcr.io/aiidateam/aiida-core-with-services:latest",
```

Better to use image from ghcr to avoid hitting dockerhub rate limit.